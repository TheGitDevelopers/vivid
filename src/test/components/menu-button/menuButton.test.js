import React from 'react';
import renderer from 'react-test-renderer';
import MenuButton from '../../../components/common/menu-button/menuButton';


describe('MenuButton tests:', () => {
  /* Test mockups */
  const tree = renderer
    .create(<MenuButton />);

  /* All component tests should be listed here */
  it('renders correctly', () => {
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
