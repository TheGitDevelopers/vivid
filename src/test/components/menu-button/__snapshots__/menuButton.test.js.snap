// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuButton tests: renders correctly 1`] = `
<View
  style={
    Object {
      "justifyContent": "center",
      "marginRight": 20,
      "position": "relative",
    }
  }
>
  <View
    style={
      Object {
        "alignSelf": "center",
        "backgroundColor": "#D6D6D6",
        "borderRadius": 50,
        "height": 40,
        "opacity": 0,
        "position": "absolute",
        "transform": Array [
          Object {
            "scaleX": 0,
          },
          Object {
            "scaleY": 0,
          },
        ],
        "width": 40,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "height": 24,
          "justifyContent": "center",
          "position": "relative",
          "transform": Array [
            Object {
              "rotate": "0deg",
            },
          ],
          "width": 24,
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "#242424",
            "borderRadius": 3,
            "height": 3,
            "position": "absolute",
            "top": 2,
            "transform": Array [
              Object {
                "translateY": 0,
              },
              Object {
                "rotate": "0deg",
              },
            ],
            "width": 24,
          }
        }
      />
      <View
        style={
          Object {
            "backgroundColor": "#242424",
            "borderRadius": 3,
            "height": 3,
            "opacity": 1,
            "position": "absolute",
            "right": 0,
            "top": 10,
            "width": 18,
          }
        }
      />
      <View
        style={
          Object {
            "backgroundColor": "#242424",
            "borderRadius": 3,
            "height": 3,
            "position": "absolute",
            "top": 18,
            "transform": Array [
              Object {
                "translateY": 0,
              },
              Object {
                "rotate": "0deg",
              },
            ],
            "width": 24,
          }
        }
      />
    </View>
  </View>
</View>
`;
